src/main/java/com/zaxxer/nuprocess/NuProcessBuilder.java

```diff
@@ -310,4 +310,26 @@ private String[] prepareEnvironment()
 
       return env;
    }
+
+   static int getGlobalBufferSizeSetting()
```

Comment:
I don't think this should be here. I'd put it on `NuProcess` instead. If a point comes where the `NuProcessBuilder` grows the ability to control the buffer size per-process, `NuProcess.BUFFER_CAPACITY` would still not be able to reflect that (since it's a static constant). I suspect if that ever became possible we'd need to deprecate that constant and either remove it or rename it (e.g. `DEFAULT_BUFFER_CAPACITY`). Regardless, though, it still wouldn't need to look at the builder to know what that value is.

Reply:
Yeah, maybe we put this in Constants class?

Reply:
I also first thought about putting it in NuProcess but it is an interface. Not a big fan of implementation in interfaces. Constants feels good to me. But if you like. No problem for me to put something in NuProcess.

Reply:
For more fine grained control I'd put it in the `NuProcessBuilder`.

`NuProcessBuilder(....).setStdoutBufferCapacity(4092).setStdinBufferCapacity(32000).setStdErrBufferCapacity(16000);`